lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -775,11 +775,11 @@
     </datatype>
     <datatype extension="pdbqt" type="galaxy.datatypes.molecules:PDBQT" display_in_upload="true"/>
     <datatype extension="pqr" type="galaxy.datatypes.molecules:PQR" display_in_upload="true" />
-    <datatype extension="cell" type="galaxy.datatypes.molecules:Cell" display_in_upload="true"/>
-    <datatype extension="cif" type="galaxy.datatypes.molecules:CIF" display_in_upload="true"/>
+    <datatype extension="cell" type="galaxy.datatypes.molecules:Cell" display_in_upload="true" description="CASTEP input file contains geometry information, including cell parameters, atomic coordinates, and k-point coordinates and weights. The file consists of lines containing keywords and data blocks, and associated values"/>
+    <datatype extension="cif" type="galaxy.datatypes.molecules:CIF" display_in_upload="true" description="CIF or Crystallographic Information File is the standard format for storing crystallographic structural data" description_url="https://www.iucr.org/resources/cif"/>
     <datatype extension="xyz" type="galaxy.datatypes.molecules:XYZ" display_in_upload="true"/>
     <datatype extension="extxyz" type="galaxy.datatypes.molecules:ExtendedXYZ" display_in_upload="true"/>
```

Comment:
I'd probably use https://wiki.fysik.dtu.dk/ase/ase/io/formatoptions.html#extxyz for extxyz, since we're actually using ASE under the hood and it's a pretty descriptive reference (I don't find https://github.com/libAtoms/extxyz that useful).

Could use https://en.wikipedia.org/wiki/XYZ_file_format for xyz (what source can be more reliable than Wikipedia). Even saying something like "Basic format for atomic positions in a structure" would be something.